examples/extend-express-app/keystone.ts

```diff
@@ -23,18 +34,8 @@ export default config<TypeInfo>({
         Keystone schema and return the results as JSON
     */
     extendExpressApp: (app, commonContext) => {
-      app.use('/rest', async (req, res, next) => {
```

Comment:
We don't want to call `withRequest` for other HTTP requests, except for `GET /rest/tasks`, thereby we shouldn't use `app.use`.